src/main/java/io/dropwizard/kafka/KafkaAdminClientFactory.java

```diff
@@ -15,19 +15,19 @@
 import org.apache.kafka.common.metrics.MetricsReporter;
 import org.apache.kafka.common.metrics.Sensor;
 
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
```

Comment:
Imports are not ordered correctly.

Reply:
Not ordered correctly according to what?

It seems like your IDE import configuration is different than the configuration we use when this project was written. When contributing to a project, you should make an effort to stay consistent with the project's existing conventions and style.